lib/msal-common/src/authority/Authority.ts

```diff
@@ -507,14 +507,24 @@ export class Authority {
          * hardcoded list of metadata
          */
 
+        const openIdConfigurationEndpoint = this.defaultOpenIdConfigurationEndpoint;
+        this.logger.verbose(`Authority.getEndpointMetadataFromNetwork: attempting to retrieve OAuth endpoints from ${openIdConfigurationEndpoint}`);
+
         try {
             const response =
                 await this.networkInterface.sendGetRequestAsync<OpenIdConfigResponse>(
-                    this.defaultOpenIdConfigurationEndpoint,
+                    openIdConfigurationEndpoint,
                     options
                 );
-            return isOpenIdConfigResponse(response.body) ? response.body : null;
+            const isValidResponse = isOpenIdConfigResponse(response.body);
+            if (isValidResponse) {
+                return response.body;
+            } else {
+                this.logger.verbose(`Authority.getEndpointMetadataFromNetwork: could not parse response as OpenID configuration`);
```

Comment:
We should have telemetry if a network call fails, adding a data point in addition to the verbose logging would be good.

Reply:
That's a good point, thank you. Done.